src/cache.rs

```diff
@@ -13,17 +13,17 @@ use dusk_pki::PublicSpendKey;
 use dusk_plonk::prelude::BlsScalar;
 use dusk_wallet_core::Store;
 use phoenix_core::Note;
-use rocksdb::{Options, WriteBatch, DB};
+use rocksdb::{DBWithThreadMode, MultiThreaded, Options};
 
 use crate::{error::Error, store::LocalStore, MAX_ADDRESSES};
 
+type DB = DBWithThreadMode<MultiThreaded>;
+
 /// A cache of notes received from Rusk.
 ///
 /// path is the path of the rocks db database
 pub(crate) struct Cache {
     db: DB,
-    // Persist writes atomically
```

Comment:
Can you explain why this has been removed? (I mean the atomically)

Reply:
We kept the persist because we were first porting microkelvin to rocksdb and wanted to keep the API same. 

Now we use a multithreaded database which doesn't need a persist call which better suits our need since the network call is async.